packages/dnote/dnote-server.subpackage.sh

```diff
@@ -0,0 +1,2 @@
+TERMUX_SUBPKG_DESCRIPTION="dnote server"
+TERMUX_SUBPKG_INCLUDE="bin/dnote-server, share/doc/${TERMUX_PKG_NAME}/AGPLv3.txt"
```

Comment:
This field shouldn't be comma separated, remove the commas and dnote / dnote-server should be put in the subpackages